easybuild/easyconfigs/l/LAMMPS/LAMMPS-patch_4Feb2020-foss-2019a-Python-3.7.2-CUDA-10.1.105-kokkos.eb

```diff
@@ -0,0 +1,121 @@
+name = 'LAMMPS'
+version = 'patch_4Feb2020'
```

Comment:
I would use a more sensible version, like `20200204`...

Thoughts on that @ocaisa?

Reply:
But this corresponds with their versioning on github archive? You would need to add another variable/change additional things in sources when updating toolchain in future?

Reply:
Or you don't like `patch_4Feb2020` names because of sorting?

Reply:
Exactly, it's not a sensible version at all to me...

Reply:
Well it is the upstream version number for it. Yeah, they have no clue on how to do versioning...

Reply:
On the other hand I would stay away from the patch releases and only do the stable ones to start with. At least the don't start with "patch_"

Reply:
But the stable releases nowadays start with "stable_:". So we should probably drop that part for the version string and handle it some way. The last non-lammps-easyblock PR for LAMMPS (i.e. CMakeMake based one) should be looked at.

Reply:
@akesandgren So, the `patch_*` releases are not bug fix releases? Or what are you saying?

Reply:
I don't know really. There are stable releases and then the ones they call patch...
And they've been popping up too often lately for me to trust them. considering how slow they where before.

Reply:
Patches are apparently RCs(release-candidates) - latest patch comment starts with: 

> This is a release candidate for the Winter 2019/2020 stable release of LAMMPS.

Let me sum options:
1) choose custom versioning scheme (for example `20200204`)
   - good for sorting
   - can't use `sources = ['%(version)s.tar.gz']` - needs translation
2) use stable_ releases and version would be just `4Feb2020`
   - `sources = [stable_'%(version)s.tar.gz']`
   - you can manually change sources string for patch release if needed
3) keep current versioning

I'm in favor of option 2, since it will have some kind of "date" version. But if someone will change the version to try the patch release they will need to change sources string as well, which could be source of frustration/confusion.

Reply:
I don't like the f*cked up versioning, but I guess diverging from it will only lead to confusion.

So let's go with option 2?

Reply:
Yup, I've implemented it that way. And share the same feeling about versioning...